gnomad_qc/v4/resources/basics.py

```diff
@@ -103,7 +121,14 @@ def _remove_ukbb_dup_by_index(
     vds = hl.vds.VariantDataset(rd, vd)
 
     # Filter withdrawn samples from the VDS
-    withdrawn_ids = withdrawn_ids + ids_to_remove + hl.eval(dup_ids)
+    withdrawn_ids = (
+        withdrawn_ukbb_samples
+        + withdrawn_26041_from_sample_map
```

Comment:
I think I'm not following all of this, so we might need some extra notes to make sure it's clear. So does this mean there are other IDs in the 26041 application that are not in the 31063 application, but somehow are also not captured by the `withdrawn_ukbb_samples`?